content/en/docs/contributing/coding-conventions.md

```diff
@@ -50,3 +50,15 @@ import (
 
 Once this manual split of standard library, external and internal imports has been made, it will be
 enforced automatically by `goimports` when executed in the future.
+
+## UK vs. US spelling
+
+For the sake of consistency, we settled on using the US spelling for the
+documentation in https://cert-manager.io as well as within the cert-manager
+codebase.
+
+For example:
+
+- "authorise" should be spelled "authorize",
+- "honour" should be spelled "honor",
+- "standardise" should be spelled "standardize".
```

Comment:
Rather than looking to give examples here, would it be helpful instead link to a more comprehensive list, e.g. https://wiki.ubuntu.com/EnglishTranslation/WordSubstitution ?

That list is from "translating" en_US -> en_GB, but I think it would work to illustrate the idea.

Reply:
I like this suggestion, since it means the spell-checker won't have to know to ignore the examples.